core/capabilities/syncer.go

```diff
@@ -25,19 +25,29 @@ import (
 	p2ptypes "github.com/smartcontractkit/chainlink/v2/core/services/p2p/types"
 )
 
+type reader interface {
+	state(ctx context.Context) (state, error)
+}
+
 type registrySyncer struct {
 	peerWrapper  p2ptypes.PeerWrapper
 	registry     core.CapabilitiesRegistry
 	dispatcher   remotetypes.Dispatcher
+	stopCh       services.StopChan
 	subServices  []services.Service
 	networkSetup HardcodedDonNetworkSetup
+	reader       reader
 
 	wg   sync.WaitGroup
 	lggr logger.Logger
 }
 
 var _ services.Service = &registrySyncer{}
 
+var (
+	defaultTickInterval = time.Duration(12 * time.Minute)
```

Comment:
Small nit: do you mean 12 seconds here (every block on Ethereum)

Reply:
Good catch :)